### PR TITLE
changed logger to write to stdout if the log file is read only

### PR DESCRIPTION
**Pull request by roironn**

### CODE REVIEW

File: packages/logging/src/internal/log-file.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+*/
+import fs from 'fs'
+import path from 'path'
+
+const validateAccess = (filename: string): boolean => {
+  try {
+    // eslint-disable-next-line no-bitwise
+    fs.accessSync(filename, fs.constants.R_OK | fs.constants.W_OK)
+    return true
+  } catch (err) {
```
Comment (netama) [inline packages/logging/src/internal/log-file.ts:24]: nit: might be worth logging the error for debugging?

**Review by netama — APPROVED** [review]
LGTM. can you please add the ticket number to the title?

**Review by royra — CHANGES_REQUESTED** [review]
Sorry for butting in, but the loggers package should work on the browser - no fs. Also this is specific behavior for the client and belongs there.

**Comment by roironn** [comment]
> Sorry for butting in, but the loggers package should work on the browser - no fs. Also this is specific behavior for the client and belongs there.

Then why does is natively support a SALTO_LOG_FILE env variable?

**Comment by roironn** [comment]
> Sorry for butting in, but the loggers package should work on the browser - no fs. Also this is specific behavior for the client and belongs there.

Ahm...

https://github.com/salto-io/salto/blob/19b53505fd448d8d7f448082b29e4701b836d4e3/packages/logging/src/internal/pino.ts#L18

**Comment by royra** [comment]
> > Sorry for butting in, but the loggers package should work on the browser - no fs. Also this is specific behavior for the client and belongs there.
> 
> Then why does is natively support a SALTO_LOG_FILE env variable?
> Ahm...
> 
> https://github.com/salto-io/salto/blob/19b53505fd448d8d7f448082b29e4701b836d4e3/packages/logging/src/internal/pino.ts#L18

Supporting fs is not like requiring it.
Looks like fs.createWriteStream is ported/mocked at the browser in CRA by chance.
Nonetheless the plain use of it looks like a bug in the logging module, so it's not a precedent for more bugs. 
I don't think you'll have such luck with stuff like `fs.accessSync`.

I think the other point re: separation of concerns is more important than the technical one. IMO `@salto-io/logging` should stand on its own regardless of its uses by core/saas. If core needs this specific behavior it can easily be implemented there.

**Review by royra — APPROVED** [review]
After v-f2f discussion: Approving, need to make sure it doesn't break saas ui.

**Pull request merged**